bindings/src/main/scala/chrome/contextMenus/ContextMenus.scala

```diff
@@ -0,0 +1,34 @@
+package chrome.contextMenus
+
+import chrome.ChromeAPI
+import chrome.contextMenus.bindings.{CreateProperties, MenuInfo, UpdateProperties}
+import chrome.permissions.APIPermission
+import chrome.tabs.bindings.Tab
+
+import scala.scalajs.js
+import scala.scalajs.js.|
+import scalajs.js.JSConverters._
+
+/**
+  * Created by antonkulaga on 07/06/16.
+  */
+object ContextMenus  extends ChromeAPI
+{
+  val ContextMenus = APIPermission("contextMenus", "permission to add items to content menus")
+
+  def addOnClick(callback: (MenuInfo, Tab) => Unit): Unit = bindings.OnMenuClicked.addListener(callback)
+
+  val requiredPermissions: Set[APIPermission] = Set(ContextMenus)
+
+  def create(createProperties: CreateProperties): String | Int = bindings.ContextMenus.create(createProperties)
+
+  def create(id: String, title: String, contexts: List[String] = List("all")): String | Int = create(bindings.CreateProperties(id, title, contexts = js.Array(contexts:_*)))
+
+  def update(id: String | Int, properties: UpdateProperties): Unit = bindings.ContextMenus.update(id, properties )
+
+  def remove(menuItemId: String | Int, callback: ()=>Unit): String | Int = bindings.ContextMenus.remove(menuItemId, callback)
```

Comment:
i usually map callback apis to `Future` in the non binding version. See https://github.com/lucidd/scala-js-chrome/blob/master/bindings/src/main/scala/chrome/alarms/Alarms.scala#L48 for an example.